modules/ROOT/partials/configuration/text_color.adoc

```diff
@@ -10,6 +10,7 @@ This option allows for specifying the number of columns for text color grids. Th
 
 By default, the number of rows and columns is dependent of the number of colors specified using xref:color_map[`+color_map+`]. The dimensions of the grid will be calculated by {productname} to keep the grid a square or a rectangle with a minimum of 5 columns.
 
+// TODO: Do we want to remove this now?
 NOTE: The `+textcolor_cols+` and `+textcolor_rows+` properties have been removed in {productname}{nbsp}5.
```

Comment:
TODO here. It looks like this needs to be resolved before we can merge this? To me we should remove this as it'd only be useful when migrating from 4 -> 5.

- [ ] Resolve TODO

Reply:
I have just removed it for now as it probably doesn't make sense to keep it. I can put it back if others think it is still required